roottest/root/io/TFile/CMakeLists.txt

```diff
@@ -1,9 +1,30 @@
-#-------------------------------------------------------------------------------
-#
-# Placeholder file to translate the tests to the new CTest system. Meanwhile we
-# define a CTest test that calls 'make' in ${CMAKE_CURRENT_SOURCE_DIR}
-#
-#-------------------------------------------------------------------------------
-if(geom)
-  ROOTTEST_ADD_OLDTEST()
+ROOTTEST_ADD_TEST(assert_tcontext_thread_hoping
+                  MACRO assert_tcontext_thread_hoping.cxx
+                  OUTREF assert_tcontext_thread_hoping.ref)
+
+
+if(CMAKE_SIZEOF_VOID_P EQUAL 8)
```

Comment:
The existing test was not conditional.   Why remove the 32 bit test?

Reply:
I forgot that we still have Windows 32 bit testing. 
I will add it back now

Reply:
Now I remember - test was doing nothing in case of 32 bit. Check - it starts with  `#ifdef R__B64`
One also can see in 32-bit ref file - it is empty.

Therefore there is no any advantage to run dummy macro in Win32.

Reply:
Indeed.  That test does not make sense to run on 32bits.  Thanks.